src/protocols/vnc/settings.c

```diff
@@ -66,6 +66,7 @@ const char* GUAC_VNC_CLIENT_ARGS[] = {
     "sftp-private-key",
     "sftp-passphrase",
     "sftp-directory",
+    "sftp-keepalive",
```

Comment:
Tweaked this, and several of the other instances of keepalive to server-alive-interval.